components/Badges/badges.css

```diff
@@ -0,0 +1,91 @@
+@import "/reset.css";
+.box{
```

Comment:
give space after the class name to make code batter readable.

Reply:
ok sure